app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError("Error: Invalid command format. "
+                         "Please use 'cp file copy_file'")
+
+    with (open(parts[1], "r") as file_origin,
+          open(parts[2], "w") as file_copy):
+
+        text = file_origin.read()
+
+        while True:
+
+            if parts[1] == parts[2]:
+                raise NameError("Copy cannot have the same name!")
```

Comment:
I know it's intuitive to use NameError here, but it's wrong, you should create custom error, something like SameFileNamesError, since default name error in python used in different scenarios.